packages/wdio-globals/src/index.ts

```diff
@@ -0,0 +1,82 @@
+type SupportedGlobals = 'browser' | 'driver' | 'multiremotebrowser' | '$' | '$$'
```

Comment:
I saw `define` and `it` mentioned in earlier examples, do they need to be added here?

Reply:
I assume you mean `describe` and `it`? Those are added by Mocha and Jasmine and we don't handle them in WebdriverIO (see our issue conversation).